api/billingd/service/service.go

```diff
@@ -1019,14 +1018,34 @@ func (s *Service) reportUsage() error {
 }
 
 func (s *Service) reportCustomerUsage(ctx context.Context, cus *Customer) error {
+	summary := map[string]interface{}{
```

Comment:
Is there a way to simplify these with a single function, `getAnalyticsSummary()`?

Reply:
That could take a product and the customer object?

Reply:
was thinking the same makin coffee. will update